### PR TITLE
foxpass setup ansible

### PR DESCRIPTION
**Pull request by asakapab0i**
## Description of the change

Ansible implementation of foxpass setup

the script is only for Ubuntu 20.04 but later on when new distros and major versions are written we can then organize the tasks properly

At the time of writing, still currently testing this.

## Type of change
- [ ] Bug fix
- [x] New feature

### Testing

- [ ] Testing information has been added - test cases checklist and steps followed for testing.
- [ ] Testing screenshot(s) attached for more information.

## Security

- [ ] This PR has security related considerations.

### CODE REVIEW

File: automation/foxpass-ansible-setup/roles/foxpass-setup/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (ttheune) [inline automation/foxpass-ansible-setup/roles/foxpass-setup/.travis.yml]: lets not check in the .travis.yml file.  add it to the gitignore list.

File: automation/foxpass-ansible-setup/roles/foxpass-setup/handlers/main.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  systemd:
+    name: ssh
+    state: stopped
+
+- name: restart ssh
+  systemd:
+    name: ssh
+    state: restarted
+
+- name: restart nslcd
```
Comment (ttheune) [inline automation/foxpass-ansible-setup/roles/foxpass-setup/handlers/main.yml]: nslcd/nscd are only for Debian based distros, RHEL based distros use sssd.  Make main.yml distro agnostic please

File: automation/foxpass-ansible-setup/roles/foxpass-setup/handlers/main.yml [hunk 3]
```diff
@@ -1,25 +1,14 @@
...
+- name: specific handlers for Ubuntu
+  import_tasks: Ubuntu/main.yml
+  when: ansible_distribution == "Ubuntu"
 
-- name: restart ssh
-  systemd:
-    name: ssh
-    state: restarted
+- name: specific handlers for CentOS
+  import_tasks: Ubuntu/main.yml
```
Comment (ttheune) [inline automation/foxpass-ansible-setup/roles/foxpass-setup/handlers/main.yml]: should we be importing Ubuntu/main.yaml for CentOS?

Comment (asakapab0i) [reply automation/foxpass-ansible-setup/roles/foxpass-setup/handlers/main.yml]: No, my bad xD.

**Comment by asakapab0i** [comment]
Hi @ttheune , tested both Ubuntu and CentOS.

Ubuntu does not have any issues but with CentOS the connection is prevented by SELinux, which is expected and we have a workaround for that.

**Review by ttheune — APPROVED** [review]
looks good!  approved if tested.

**Pull request merged**